demo-apps/chaos-monkey-demo-app-unleash-toggles/src/test/java/com/example/chaos/monkey/toggledemo/component/HelloComponentTest.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2018-2024 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.example.chaos.monkey.toggledemo.component;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class HelloComponentTest {
```

Comment:
Which value adds this test? 😅 if its only to get codecov happy I would simply drop it

Reply:
But if codecov drops down, you have to merge it anyway

Reply:
yeah codecov cant stop us 👿 😄 we merge it!

![never stop](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExcnQwZWFsa2RwbmsyZWN0eWxmNzJqZmFycGJmNTlsNWMycWd1bTRxaiZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/3orif9Y5Tq35e9qTGU/giphy.gif)

Reply:
![Do-it1](https://github.com/user-attachments/assets/2379e11c-9967-4104-9d70-c3061ce7b32a)

Reply:
if you fix [that](https://github.com/codecentric/chaos-monkey-spring-boot/pull/518#discussion_r1873299022), im happy to give the approval :D